tests/spec/core/pluralize-spec.js

```diff
@@ -106,26 +106,6 @@ describe("Core - Pluralize", () => {
     ).toBeTruthy();
   });
 
-  it("does nothing if conf.pluralize is not defined", async () => {
```

Comment:
Actually,  I wonder if I can do this differently without trashing the test... I might be able to set status to unofficial, and that should stop treating this as a w3c spec.

Reply:
ok, should hopefully be good now.